swarm/network/hive.go

```diff
@@ -55,8 +56,7 @@ func NewHiveParams() *HiveParams {
 		MaxPeersPerRequest:    5,
 		KeepAliveInterval:     500 * time.Millisecond,
 		RetryInterval:         4200000000, // 4.2 sec
-		//RetryExponent:         2,
-		RetryExponent: 3,
+		RetryExponent:         3,
```

Comment:
why 3?

Reply:
i just removed the commented one. was it 2 in the first place?